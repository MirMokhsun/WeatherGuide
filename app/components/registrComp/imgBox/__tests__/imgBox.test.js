import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ImgBox from '..';

describe('ImgBox', () => {
    test('ImgBox snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<ImgBox />);
        expect(snapshot).toMatchSnapshot();
    });
});