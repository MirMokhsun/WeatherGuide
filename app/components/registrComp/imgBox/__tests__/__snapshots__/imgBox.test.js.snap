// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImgBox ImgBox snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "marginTop": 30,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../app/assets/logos/cloudy-1.png",
      }
    }
    style={
      Object {
        "marginTop": 10,
        "maxHeight": 160,
        "maxWidth": 160,
      }
    }
  />
  <Text
    style={
      Object {
        "fontFamily": "Roboto-Black",
        "fontSize": 30,
        "marginTop": 5,
      }
    }
  >
    Weather Guide
  </Text>
</View>
`;
